src/Symfony/Component/HttpKernel/DependencyInjection/RegisterControllerArgumentLocatorsPass.php

```diff
@@ -120,6 +121,21 @@ public function process(ContainerBuilder $container)
                 $args = [];
                 foreach ($parameters as $p) {
                     /** @var \ReflectionParameter $p */
+                    if (class_exists(Autowire::class) && $attr = $p->getAttributes(Autowire::class)[0] ?? null) {
```

Comment:
Is the `class_exists()` check necessary? Calling `$p->getAttributes(InvalidClass::class)` should be a safe operation.